docs/visual-basic/misc/bc30283.md

```diff
@@ -46,4 +46,4 @@ A constructor indicates that it overrides an inherited constructor. Constructors
   
 ## See Also  
  [NOT IN BUILD: Overriding Properties and Methods](http://msdn.microsoft.com/en-us/2167e8f5-1225-4b13-9ebd-02591ba90213)   
```

Comment:
This can be `[Overriding properties and methods in derived classes](~docs/visual-basic/programming-guide/language-features/objects-and-classes/inheritance-basics.md#overriding-properties-and-methods-in-derived-classes)`